src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -5,6 +5,8 @@ import React, {
   useRef,
   useState,
 } from 'react';
+import { Keyboard } from 'react-native';
+
```

Comment:
why the newline?

Reply:
I can see that in other files, we separate react-native imports from rest of the imports

Reply:
urh, nvm. We separate 3rd party imports!!